website/src/pages/dashboard.tsx

```diff
@@ -66,7 +66,7 @@ const Dashboard = () => {
         {ENABLE_CHAT && (
           <Flex direction="column" gap={4}>
             <Heading size="lg">{t("index:try_our_assistant")}</Heading>
-            <Link href="/chat" aria-label="Chat">
+            <Link href="/chat" aria-label="Chat" style={{ width: "max-content" }}>
```

Comment:
Disclamer: I am not a professional frontend developer

Isn't it more correct to put link inside a button? Wouldn't it solve the issue without additional css? Or maybe get rid of button tag at all and make it just link if possible?

Reply:
Even I thought of same, but that would require refactoring decisions, so I'm not sure what is the process and just raised a PR to fix the existing issue.

Reply:
Well, let's wait for a team member opinion on this then

Reply:
It is better to render the button as a tag, but I think its fine for now.